launcher/container_runner.go

```diff
@@ -26,12 +29,21 @@ import (
 	"google.golang.org/grpc/credentials/insecure"
 )
 
+type attestationAgent interface {
+	MeasureEvent(cel.Content) error
+	Attest(context.Context) ([]byte, error)
+}
+
 // ContainerRunner contains information about the container settings
 type ContainerRunner struct {
-	container   containerd.Container
-	launchSpec  spec.LauncherSpec
-	attestConn  *grpc.ClientConn
-	attestAgent *AttestationAgent
+	container      containerd.Container
+	launchSpec     spec.LauncherSpec
+	attestConn     *grpc.ClientConn
+	attestAgent    attestationAgent
+	tokenRefresher struct {
+		timer *time.Timer
+		done  chan bool
+	}
```

Comment:
It seems a little odd that the refresher is a member of the `ContainerRunner` struct. It seems like this should only exist inside `Run` method. After the `Run` call completes, the container is stopped, so we shouldn't need to be running a timer anymore.

This also then removes the need to have a separate `done` channel or do additional cleanup in `Close()`.